pages/common/todoist.md

```diff
@@ -0,0 +1,28 @@
+# todoist
+
+> This program will let you use Todoist from the CLI.
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
+
+- Add a high priority task, with label, project, and due date:
+
+`todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
+
+- Add the same task, in quick mode:
+
+`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{some_task}} @{{label_name}}'`
```

Comment:
```suggestion
`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{task_name}} @{{label_name}}'`
```

Should there be a space between the @

Reply:
no, the label has to be specified like "@labelname"